lib/check_for_wins.ex

```diff
@@ -30,18 +30,32 @@ defmodule CheckForWins do
     end
   end
 
-  def analyze(board) do
-    horizontal = convert_horizontal_to_row(board) |> check_row
+  def has_empty_spaces?(board) do
+    Map.values(board)
```

Comment:
This is nitpicky (which is good! 😁little things means the big things are being taken care of already 😊) but -- it's usually considered clearer/good practice to if possible only have "bare" values at the top of a pipe.
```suggestion
    board
    |> Map.values()
```

Reply:
I see -- makes it easier to see what the chain starts from. Will update!